drivers/kscan/kscan_mchp_xec.c

```diff
@@ -411,6 +418,9 @@ void polling_task(const struct device *dev, void *dummy2, void *dummy3)
 			/* Allow other threads to run while we sleep */
 			k_usleep(wait_period);
 		}
+		#ifdef CONFIG_PM_DEVICE
```

Comment:
nit, do we need to space ifdef like this?

Reply:
Ok, removed the space before #ifdef.